package.json

```diff
@@ -696,7 +696,8 @@
             "enum": [
               "Interactive",
               "Open",
-              "Add to Workspace"
+              "Add to Workspace",
+              "None"
```

Comment:
Is this value referenced in code base?

Reply:
No. 

In `specifyOpenMethod`
If value of setting is "Interactive", we prompt to ask.
Otherwise value of setting is directly returned.

Based on above result:
If Open ...
If Add To Workspace ...

For the case `None`, we simply do nothing.